docs/ocp/openshift-install.md

```diff
@@ -83,15 +82,71 @@ To start, ensure your `kubectl` or `oc` is configured to point to your OpenShift
    helm upgrade --install kagenti ./charts/kagenti/ -n kagenti-system --create-namespace -f ./charts/kagenti/.secrets.yaml
    ```
 
-## Access the UI
+## Authentication Configuration
+
+Kagenti UI now supports Keycloak authentication by default.
+
+### Prerequisites
 
-After the chart installs, you may access the UI following the instructions in the notes; the URL to UI can be found 
-running this command:
+**IMPORTANT**: Before accessing the UI, you must create a Kubernetes secret named `kagenti-ui-oauth-secret` in the `kagenti-system` namespace with the following keys:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kagenti-ui-oauth-secret
+  namespace: kagenti-system
+type: Opaque
+stringData:
+  ENABLE_AUTH: "true"
+  CLIENT_ID: "kagenti"
+  CLIENT_SECRET: "<your-keycloak-client-secret>"
+  AUTH_ENDPOINT: "https://<keycloak-route>/realms/master/protocol/openid-connect/auth"
+  TOKEN_ENDPOINT: "https://<keycloak-route>/realms/master/protocol/openid-connect/token"
+  REDIRECT_URI: "https://<ui-route>/oauth2/callback"
+  SCOPE: "openid profile email"
+```
+
+Replace:
+- `<your-keycloak-client-secret>`: The client secret from your Keycloak client
```

Comment:
some explanation or pointers on how to get the keycloak client secret would be helpful (until we have the automation in place)